api/serverless/getRepolinterResult/app.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path');
+const repolinter = require('repolinter');
+const uuidv4 = require('uuid-v4');
+const rimraf = require('rimraf');
+const fs = require('fs');
+const git = require('simple-git')();
+
+async function runRepolinter(gitLink, ruleSet) {
+  const tmpDir = await path.resolve('/tmp', uuidv4());
+  await asyncGitClone(gitLink, tmpDir).catch(error => console.error(error));
+
+  const output = await asyncRunner(tmpDir, ruleSet);
+  rimraf(tmpDir, function() {});
+  return output;
+}
+
+async function asyncRunner(tmpDir, ruleSet) {
+  return new Promise(function(resolve, reject) {
+    const output = repolinter.lint(tmpDir, [], ruleSet);
+    if (output != null) {
+      resolve(output);
+    } else {
+      reject('error');
+    }
+  });
+}
+
+async function asyncGitClone(gitLink, tmpDir) {
+  return new Promise(function(resolve, reject) {
+    git.clone(gitLink, tmpDir, error => {
+      if (!error) {
+        resolve();
+      } else {
+        reject(error);
+      }
+    });
+  });
+}
+
+exports.lintRepo = async (event, context, callback) => {
+  process.env.PATH =
+    process.env.LAMBDA_TASK_ROOT + '/tmp/bin/usr/bin:' + process.env.PATH;
+  process.env.GIT_EXEC_PATH =
+    process.env.LAMBDA_TASK_ROOT + '/tmp/bin/usr/libexec/git-core';
+  const runningConfig = JSON.parse(event.body);
+  try {
+    const ret = await runRepolinter(
+      decodeURI(runningConfig.git_link.replace(/([%])/g, '/')),
+      runningConfig.ruleSet
+    );
+    let response = {
+      statusCode: 200,
+      headers: {
+        'Access-Control-Allow-Methods':
+          'DELETE,GET,HEAD,OPTIONS,PATCH,POST,PUT',
+        'Access-Control-Allow-Origin': '*',
+      },
+      body: JSON.stringify({
+        data: ret,
+      }),
+    };
+    return response;
+  } catch (err) {
+    callback(err, null);
```

Comment:
Throw an empty return after this line (or return the callback instead). Otherwise, callback is running twice for errors. Fix this up for the other locations as well.